inst/app/app.R

```diff
@@ -0,0 +1,219 @@
+library(shiny)
+library(shinyWidgets)
+library(shinyjs)
+library(shinybusy)
+library(ggplot2)
+library(dplyr)
+library(rplanes)
+library(lubridate)
+
+# list module files and iterate sourcing them to use within the app.
+module_sources <- list.files(path = here::here("inst/app/modules"), full.names = TRUE)
+#module_sources <- list.files(path = system.file("app/modules/", package = "rplanes"), full.names = TRUE)
+sapply(module_sources, source)
+
+
+# UI SIDE ####
+
+ui <- fluidPage(
+    useShinyjs(),  # Set up shinyjs
+    titlePanel("Rplanes Explorer"),
+    sidebarLayout(position = "left",
+                  sidebarPanel(width = 3,
+                      prettyRadioButtons("choice", "Choose Dataset", choices = c("Example", "Custom"), status = "warning", inline = TRUE, icon = icon("check"), bigger = TRUE),
```

Comment:
i spoke to this in a separate comment and havent seen any changes (see https://github.com/signaturescience/rplanes/pull/94#issuecomment-1787647153) ... but again, i think we need to *not present the "Example" data by default. at the very least please switch this so that "Custom" is selected. do review the longer comment that i posted in the PR thread though.